README.md

```diff
@@ -240,6 +249,49 @@ parts into `dist` folder.
    and this release contains all the needed binaries and checksums.
 5. Remove the `draft` checkbox for the `TAG` release and manually publish it.
 
+## How to use WITH templates
+
+The `WITH` templates feature simplifies the construction and management of complex queries.
+You can try this feature in the [WITH templates playground](https://play.victoriametrics.com/select/accounting/1/6a716b0f-38bc-4856-90ce-448fd713e3fe/expand-with-exprs).
+
+The "WITH templates" section allows you to create expressions with templates that can be used in dashboards.
```

Comment:
We should mention where WITH expressions are stored. Because re-imported dashboard won't have created with templates in it.